src/Workspaces/Core/Portable/CodeStyle/CodeStyleOption.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#pragma warning disable RS0030 // Do not used banned APIs: CodeStyleOption<T>
```

Comment:
💡 Please place below `#nullable disable` when both are present. Otherwise when you open the file for a quick glance, it looks like this file is not marked with `#nullable disable`.